public/509.html

```diff
@@ -0,0 +1,143 @@
+            <li id="medium_1"><a href="/media/TV%20Shows/fandoms">TV Shows</a></li>
+            <li id="medium_476"><a href="/media/Video%20Games/fandoms">Video Games</a></li>
+            <li id="medium_9971"><a href="/media/Uncategorized%20Fandoms/fandoms">Uncategorized Fandoms</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="/menu/browse">Browse</a>
+          <ul class="menu" role="menu">
+            <li><a href="/works">Works</a></li>
+            <li><a href="/bookmarks">Bookmarks</a></li>
+            <li><a href="/tags">Tags</a></li>
+            <li><a href="/collections">Collections</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="/menu/search">Search</a>
+          <ul class="menu" role="menu">
+            <li><a href="/works/search">Works</a></li>
+            <li><a href="/people/search">People</a></li>
+            <li><a href="/bookmarks/search">Bookmarks</a></li>
+            <li><a href="/tags/search">Tags</a></li>
+          </ul>
+        </li>
+        <li class="dropdown">
+          <a href="/menu/about">About</a>
+          <ul class="menu" role="menu">
+            <li><a href="/about">About Us</a></li>
+            <li><a href="/admin_posts">News</a></li>
+            <li><a href="/archive_faqs">FAQ</a></li>
+            <li><a href="/donate">Donate or Volunteer</a></li>
+          </ul>
+        </li>
+	      <li class="search"><form accept-charset="UTF-8" action="/works/search" class="search" id="search" method="get"><div style="margin:0;padding:0;display:inline"><input name="utf8" type="hidden" value="&#x2713;" /></div>
+          <fieldset>
+            <legend>Search Works</legend>
+              <p>
+                <label class="landmark" for="site_search">Work Search:</label>
+                <input aria-describedby="site_search_tooltip" class="text" id="site_search" name="work_search[query]" type="text">
+                <span class="tip" role="tooltip" id="site_search_tooltip">tip: lex m/m (mature OR explicit)</span>
+                <span class="submit actions"><input class="button" type="submit" value="Search" /></span>
+              </p>
+            </fieldset>
+          </form>
+        </li>
+      </ul>
+      <div class="clear"></div>
+    </div>
+    <!-- END header -->
+    <div id="main" class="system error 509 maintenance">
+      <h2>Error 509 - Service unavailable</h2>
+      <h3>Exceeded maximum allowed downloads per minute.</h3>
+      <p>To combat bots, we are currently banning IP addresses that download too many works in a short time period. If you see this page repeatedly please pause at least 10 seconds between downloads. If you are banned, you will be unable to access the Archive. Access will be restored 24 hours after the ban started.</p>
+      <p>Follow <a href="http://twitter.com/ao3_status/">@AO3_Status</a> on twitter for updates.</p>
```

Comment:
I'm not sure we need this message here, since one user downloading too much isn't likely to be addressed on the @AO3_Status Twitter page. On the other hand, it never hurts to encourage people to follow! So it's up to you whether this should stay in, but if it does, can you capitalize the T in Twitter?